namespace java thrift.test

struct myI32Struct {
  1: i32 a
}
